tap/build.gradle

```diff
@@ -45,3 +54,6 @@ configurations {
   runtime.exclude group: 'net.sourceforge.jtds'
   runtime.exclude group: 'org.restlet.jee'
 }
+
+
+
```

Comment:
Minor nit: Stray blank lines added

Reply:
Oops, thanks for catching that. Should be fixed now, if build is ok I'll merge in and do a new release.
Re: the merging of the repos: I think that should be the target, I'll set this an initial investigation task for myself to evaluate what would be needed to do so